keyboards/levinson/keymaps/jyh/keymap.c

```diff
@@ -0,0 +1,232 @@
+#define SFT_ENT MT(MOD_RSFT, KC_ENT)  //  Enter   Right Shift
+#define CTL_SPC MT(MOD_LCTL, KC_SPC)  //  Space   Left Control
+#define ALT_SPC MT(MOD_LALT, KC_SPC)  //  Space   Left Alt
+#define ALT_BSP MT(MOD_LALT, KC_BSPC) //  Bkspace Left Alt
+#define HPR_TAB MT(MOD_HYPR, KC_TAB)  //  Tab     Hyper (Super+Ctrl+Alt+Shift)
+#define HPR_GRV MT(MOD_HYPR, KC_GRV)  //  `       Hyper
+#define HPR_TIL MT(MOD_HYPR, KC_TILD) //  ~       Hyper
+#define MEH_TIL MT(MOD_MEH,  KC_GRV)  //  `       Meh (Ctrl+Alt+Shift)
+#define SFT_MIN MT(MOD_LSFT, KC_MINS) //  -       Left Shift
+#define NPD_UND LT(_NUMS,    KC_UNDS) //  _       Layer Numpad
+#define FNC_PIP LT(_FUNC,    KC_PIPE) //  |       Layer Function Keys
+
+#define CTL_DEL MT(MOD_LCTL, KC_DEL)  //  Del     Left Control
+#define CTL_QOT MT(MOD_RCTL, KC_QUOT) //  '       Right Control
+#define CTL_BSL MT(MOD_RCTL, KC_BSLS) //  \       Right Control
+#define CTL_PIP MT(MOD_RCTL, KC_PIPE) //  |       Right Control
+
+// Redefine for KC_KEYMAP
+#define KC_CTL_BSL CTL_BSL
+#define KC_CTL_PIP CTL_PIP
+#define KC_CTL_DEL CTL_DEL
+#define KC_HPR_TIL HPR_TIL
+#define KC_HPR_GRV HPR_GRV
+
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+
+/* Qwerty
+ *
+ * ,-----------------------------------------.    ,-----------------------------------------.
+ * | Tab  |  Q   |  W   |  E   |  R   |  T   |    |  Y   |  U   |  I   |  O   |  P   | Bksp |
+ * |------+------+------+------+------+------|    |------+------+------+------+------+------|
+ * | Esc  |  A   |  S   |  D   |  F   |  G   |    |  H   |  J   |  K   |  L   |  ;   |  '   |
+ * |------+------+------+------+------+------|    |------+------+------+------+------+------|
+ * |  -   |  Z   |  X   |  C   |  V   |  B   |    |  N   |  M   |  ,   |  .   |  /   | Enter|
+ * |------+------+------+------+------+------|    |------+------+------+------+------+------|
+ * | Numpd| Meh  | Meta | Alt  | Lower| Space|    | Space| Raise| Alt  | Meta | Hyper| Func |
+ * `-----------------------------------------'    `-----------------------------------------'
+ *
+ * with Hold (Emacs-oriented)
+ * ,-----------------------------------------.    ,-----------------------------------------.
+ * | Hyper|      |      |      |      |      |    |      |      |      |      |      |      |
+ * |------+------+------+------+------+------|    |------+------+------+------+------+------|
+ * | Ctrl |      |      |      |      |      |    |      |      |      |      |      | Ctrl |
+ * |------+------+------+------+------+------|    |------+------+------+------+------+------|
+ * | Shift|      |      |      |      |      |    |      |      |      |      |      | Shift|
+ * |------+------+------+------+------+------|    |------+------+------+------+------+------|
+ * |      |      |      |      |      |      |    |      |      |      |      |      |      |
+ * `-----------------------------------------'    `-----------------------------------------'
+ */
+				[_QWERTY] = KEYMAP(
```

Comment:
`KEYMAP` is no longer used for this keyboard

Please replace this with `LAYOUT` or `LAYOUT_ortho_4x12`

```suggestion
				[_QWERTY] = LAYOUT_ortho_4x12(
```